arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import { useGettext } from "vue3-gettext";
+import Button from "primevue/button";
+
+const { $gettext } = useGettext();
+
+const props = defineProps<{
+    editorMax: boolean;
+}>();
+
+const emit = defineEmits(["maximize", "side", "close"]);
+
+const toggleSize = () => {
+    if (props.editorMax) {
+        emit("maximize");
+    } else {
+        emit("side");
+    }
+};
+</script>
+
+<template>
+    <div class="header">
+        <div>
+            <h2>{{ $gettext("Scheme Editor Tools") }}</h2>
```

Comment:
Same as in the other PR, it's likely this should be h3. (h1 = lingo, h2 = scheme name). h1 is being fixed in #91.